flaskapp.py

```diff
@@ -266,7 +274,11 @@ def __init__(self):
             'record:mx': [],
             'record:srv': [],
             'record:naptr': [],
-            'record:txt': []
+            'record:txt': [],
+            'nsgroup:delegation': [],
+            'nsgroup:forwardingmember': [],
+            'nsgroup:forwardstubserver': [],
+            'nsgroup:stubmember': []
```

Comment:
Leave a trailing comma for cleaner diffs in the future:

```suggestion
            'nsgroup:stubmember': [],
```